move.go

```diff
@@ -0,0 +1,20 @@
+package pokemonbattlelib
+
+import "fmt"
+
+//A type for moves
```

Comment:
Needs to be slightly more descriptive
```suggestion
// Represents a Pokemon's move. Moves can deal damage, heal the user or allies, or cause status effects.
```